Mathlib/Data/PFunctor/Univariate/Basic.lean

```diff
@@ -24,26 +24,26 @@ An element of `P α` is a pair `⟨a, f⟩`, where `a` is an element of a type `
 `f : B a → α`. Think of `a` as the shape of the object and `f` as an index to the relevant
 elements of `α`.
 -/
-@[pp_with_univ]
+@[pp_with_univ, nolint checkUnivs]
 structure PFunctor where
```

Comment:
For clarity, I think
```
structure PFunctor where
Comment
For clarity, I think
```
structure PFunctor.{uA,uB} where
  /-- The head type -/
  A : Type uA
  /-- The child family of types -/
  B : A → Type uB
```
would be clearer.

Can you use a similar convention below?